subcommittees/local-group/local_regional_mentor_communities.md

```diff
@@ -0,0 +1,46 @@
+### Prospectus for Local/Regional Mentor/Community groups
```

Comment:
This content should be in `subcommittees/local-group/README.md ` (which currently has a bunch of FIXMEs, etc.).